src/validators/ResponseValidator.js

```diff
@@ -13,8 +13,23 @@ function responseValidator(response, data) {
     let bodyValidationResult = true, bodyValidationErrors = [],
         headersValidationResult = true, headersValidationErrors = [];
     if (bodySchema) {
-        const validator = bodySchema.validate ? bodySchema
-            : bodySchema[data.headers['Content-Type'] || DEFAULT_CONTENT_TYPE];
+        let validator;
+        if (bodySchema.validate) {
+            validator = bodySchema;
+        } else {
+            const responseContentType = data.headers['Content-Type'] || data.headers['content-type'] || DEFAULT_CONTENT_TYPE;
+            validator = bodySchema[responseContentType];
+            if (!validator) {
+                this.errors = [{
+                    message: `No schema defined for response content-type "${
```

Comment:
you already have `data.headers['Content-Type'] || data.headers['content-type'] || DEFAULT_CONTENT_TYPE` value in `responseContentType` :)

Reply:
Oh, that's funny, I noticed the duplication, restructured code a bit, and then completely forgot to use the new variable 😂 .
Thanks for noticing, fixed.

Reply:
np :)